src/jvmMain/kotlin/com/charleskorn/kaml/YamlOutput.kt

```diff
@@ -52,6 +52,9 @@ internal class YamlOutput(
 ) : AbstractEncoder(), AutoCloseable {
     private val settings = DumpSettings.builder()
         .setIndent(configuration.encodingIndentationSize)
+        .setIndicatorIndent(configuration.sequenceBlockIndent)
```

Comment:
Does SnakeYAML validate this value (`setIndicatorIndent`) to ensure that the resulting YAML is valid?

Reply:
Yep, SnakeYAML validates that it must be non-negative. I'll add a comment to clarify